libs/components/colorpicker/src/lib/modules/colorpicker/types/colorpicker-hsla.ts

```diff
@@ -1,26 +1,26 @@
 // spell-checker:ignore hsla,colorpicker
 /**
- * Specifies colors as a combination of hue, saturation, and lightness with an alpha
+ * Colors as a combination of hue, saturation, and lightness with an alpha
```

Comment:
```suggestion
 * Colors specified as a combination of hue, saturation, and lightness with an alpha
```
Same suggestion as before about sticking with `Specifies colors as...` here.